### DISCUSSION TITLE
Menu for overfloatingnotification icons like in windows

### DISCUSSION THREAD

**Discussion by uvulpos** [linuxmint/discussions#226] [Ideas]
Greetings 👋🏻,
My Notification icons for different apps increase more and more over some time and at some point it conflicts with the open apps or currently the notch from my MB pro limits the list a lot. Now that I want to switch back to Linux, I would like to prevent this kind of behavior in my distro. If this is an easy fix, I would be willing to implement it myself with a little help, but in case this is a more difficult task to do, please feel free to help me 🙂 

<img width="466" alt="Screenshot 2023-11-22 at 16 59 14" src="https://github.com/linuxmint/discussions/assets/53957681/4d8478e7-29c4-44f5-b83d-715e614f5522">

Windows fixed that issue already years ago. I don't use windows anymore for years, so this is a picture I found on Google

**Comment by rcalixte** [comment]
There was a similar applet [here](https://cinnamon-spices.linuxmint.com/applets/view/169) that appears to need updates for newer versions of Cinnamon. The source code lives [here](https://github.com/linuxmint/cinnamon-spices-applets/tree/master/show-hide-applets@mohammad-sn). If this applet seems up your alley, you can take a look through the code and try to refactor it to a working state. You can look through the code for other applets or reach out with questions. Pull requests are welcome!

cc If @mohammad-sn is looking for a new maintainer for this.

**Comment by uvulpos** [comment]
It would cure the symptoms but not solve the problem really. Also, imo this should be a native feature for all DEs. I'm wondering, why other distros or macOS do not address this kind of limitation. Is this such an edge case?

**Reply by Reno-Sifana** [reply]
I Don't know why. But in KDE Plasma 5 have natively "Show Hidden Icons" (Next to time and date) long time ago. 
![Screenshot_20231205_233237](https://github.com/linuxmint/discussions/assets/68702583/07380231-53d0-40b2-8513-89a0e09909ec)

**Reply by uvulpos** [reply]
<img alt="pepe clap" width="250px" src="https://github.com/linuxmint/discussions/assets/53957681/18681b32-1a72-4b05-ba7e-1e07ebcf0c4a"><br>
I'm not a huge KDE fan, for me, it feels bloated at some points, that's why I like cinnamon, but that's exactly what I wanted!
